### PR TITLE
Add bulk support to people.list()

### PR DESCRIPTION
**Pull request by adamweeks**
Includes changes from #653 
Limit is set at 10 people per request due to hydra limitation

### CODE REVIEW

File: packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      service: `hydra`,
+      resource: `people/?id=${ids}`
+    });
+  }
+});
+
+// /**
+//  * @class
+//  */
+// const PersonUUIDBatcher = SparkPlugin.extend({
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js]: ?

File: packages/node_modules/@ciscospark/plugin-people/src/people.js [hunk 2]
```diff
@@ -102,8 +107,57 @@ const People = SparkPlugin.extend({
...
+   *   });
+   *   // => success
    */
   list(options) {
+    if (Array.isArray(options)) {
+      const peopleIds = options;
+      return Promise.all(
+        peopleIds.map((personId) => this.batcher.request(personId))
+      );
+      // return this.request({
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/plugin-people/src/people.js]: ?

File: packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js (hunk 1 above)
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js]: I'm pretty sure this was an "OMG THIS IS WORKING!" commit. Will update.

**Comment by adamweeks** [comment]
@ianwremmel commented lines removed.

**Comment by ianwremmel** [comment]
rebase off the other PR with the fixed comment messages, then it's good to go.

**Comment by adamweeks** [comment]
@ianwremmel rebased.

**Pull request merged**